File: 2264 largest-3-same-digit-number-in-string/solution.cpp
class Solution {
public:
    string largestGoodInteger(string num) {
        string  ans[] = {
            "999","888","777","666","555","444","333","222","111","000"
        };
        for(string s:ans) {
            if(num.contains(s)) return s;
        }
        return "";
    }
};

File: 2264 largest-3-same-digit-number-in-string/solution.rs
impl Solution {
    pub fn largest_good_integer(num: String) -> String {
        let ans: [&str; 10] = ["999","888","777","666","555","444","333","222","111","000"];
        for &c in ans.iter() {
            if num.contains(c) {
                return c.to_string();
            }
        }
        "".to_string()
    }
}

